hedera-node/src/main/java/com/hedera/services/txns/contract/operation/HederaCallCodeOperation.java

```diff
@@ -41,6 +47,12 @@ public OperationResult execute(MessageFrame frame, EVM evm) {
 			return new OperationResult(
 					Optional.of(cost(frame)), Optional.of(HederaExceptionalHaltReason.INVALID_SOLIDITY_ADDRESS));
 		}
+		final var accountId = EntityIdUtils.accountParsedFromSolidityAddress(account.getAddress().toArray());
+		if (!sigsVerifier.allRequiredKeysAreActive(Set.of(accountId))) {
```

Comment:
Instead of directly going to the `sigVerifier`, shouldn't we first check that the account has `receiverSigRequried=true`?

Reply:
It is done inside: `keyRequirement`